lib/Scheduler/bb_spill.cpp

```diff
@@ -873,6 +876,11 @@ InstCount BBWithSpill::UpdtOptmlSched(InstSchedule *crntSched,
     enumBestSched_->Copy(crntSched);
     bestSched_ = enumBestSched_;
   }
+  
+  if (IsSecondPass())
+  {
+    return optmlSpillCost_;
+  }
```

Comment:
Remove braces on simple if statements. https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements